app/Console/Kernel.php

```diff
@@ -15,6 +15,7 @@ protected function schedule(Schedule $schedule)
         // $schedule->command('websockets:clean')->daily();
 
         $schedule->command('horizon:snapshot')->everyFiveMinutes();
+        $schedule->command('queue:prune-batches --hours=48 --unfinished=72 --cancelled=72')->daily();
```

Comment:
nit: It may be worth adding `onFailure()` logic here for diagnostics purposes just in case something goes awry.

Reply:
I'm not sure we're that interested in failures of the cleanup command 🤔 
it's a default framework command that is expected to work as documented.
am i missing something?